scene/gui/graph_edit.cpp

```diff
@@ -730,7 +733,7 @@ void GraphEdit::set_selected(Node *p_child) {
 void GraphEdit::_gui_input(const Ref<InputEvent> &p_ev) {
 
 	Ref<InputEventMouseMotion> mm = p_ev;
-	if (mm.is_valid() && (mm->get_button_mask() & BUTTON_MASK_MIDDLE || (mm->get_button_mask() & BUTTON_MASK_LEFT && Input::get_singleton()->is_key_pressed(KEY_SPACE)))) {
+	if (mm.is_valid() && (mm->get_button_mask() & BUTTON_MASK_RIGHT || mm->get_button_mask() & BUTTON_MASK_MIDDLE || (mm->get_button_mask() & BUTTON_MASK_LEFT && Input::get_singleton()->is_key_pressed(KEY_SPACE)))) {
```

Comment:
I don t think anywhere in Godot you do this with RMB, so adding it here is probably not a good idea

Reply:
What do you mean?

Reply:
Developer feedback is that RMB should drag. 

\<BrahRah>The RMB seems to have 0 functionality atm. there should be at least a pop up menu. The mouse plays a huge role in Visual Coding and thus needs to be implemented really really well. - https://github.com/godotengine/godot/issues/19862

Reply:
I agree with @reduz and @vnen, it's inconsistent with other editor plugins and the functionality is already available with MMB and LMB + Space.

RMB should be kept for another meaningful operation.